examples/simple-example/README.md

```diff
@@ -35,3 +35,67 @@ miranum-core. The project consists of three modules:*
 Now you can simply start the process instance and view the log in the application console.
 
 <p align="right">(<a href="#top">back to top</a>)</p>
+
+### Generate element templates
```

Comment:
⚒️ Please specify that this feature is currently only available for Camunda Platform 7 right a the top.

Reply:
In the current implementation a Bean is created during the application startup, which is responsible for scanning the annotations of the current project and generating the element-templates. 
A build script is able to execute code as well, but since there is no application to scan (at this point), it won't work with our implementation.